pulseapi/profiles/models/categories.py

```diff
@@ -59,3 +63,80 @@ class ProgramYear(models.Model):
 
     def __str__(self):
         return self.value
+
+
+class UserProfileProgramParticipation(models.Model):
```

Comment:
It feels like there should be a better name for this thing =D

Reply:
yeah I'm not good at naming bridge tables. Any suggestions?

Reply:
Thinking about it some more, this tuple basically literally encodes a `Cohort` record